grails-app/conf/Config.groovy

```diff
@@ -1209,7 +1209,7 @@ openboxes.typeahead.delay = 300
 openboxes.typeahead.minLength = 3
 
 // Allow system administrators to disable refresh on startup
-openboxes.refreshAnalyticsDataOnStartup.enabled = true
+openboxes.refreshAnalyticsDataOnStartup.enabled = false
```

Comment:
What is the reason for disabling this?

Reply:
@jmiranda this was a suggestion for development purposes. This is the WIP version, and @hdsldv will remove it in the final version

Reply:
reverted

Reply:
@awalkowiak @hdsldv no worries. i was just wondering if there was some kind of conflict with the changes we were making. 

A better approach for local development environments would be to create ~/.grails/openboxes-config.groovy and add the property there.